### DISCUSSION TITLE
[Feature] Show Tagged frame and bounding boxes when editing faces of a video asset.

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6014] [Feature Request]
### The feature

I have a hard time finding out who is tagged where when editing faces of a video.
Current User Experience:
- Open a video
- Open side pane
- Trigger "edit faces" mode 
- If the persons are already named, you see their "feature pic", not the bounding box within the current asset
- So you have to untag the person -> Create new person, to be able to understand which face we're talking about.
- Then, if the real person is already named, you have to "Undo reassign -> Reassign to the name face"

Suggestions:
- Force pause the video when entering edit mode. Rewind to beginning.
- Make sure the correct frame is displayed (Maybe it's already the case), ie the one that matches the facebox displayed in the side pane
- Highlight the facebox on this video frame when mousing over on the side pane (The opposite would be ideal too: highlight the face on the side pane when mousing over the facebox on the video frame)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Face recognition for videos is (currently) always based on the thumbnail, which is the first frame.
